pagopa/io-app
# chore: [IOPAE-1148] Add Institutions search

## Short description
This PR allows to search for national and local institutions.

<details open><summary>details</summary>

<p>

| search |
| - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/f6d3af6e-fb3e-4c81-819c-5637ec7ef828" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added the `SearchScreen`in order to search for national and local institutions
- added the generic component `EmptyState`
- added `@shopify/flash-list` package
- updated locales

## How to test
Using `io-dev-api-server`, Navigate to the services tab and tap the search button. Try the search and check that everything works correctly.
